data/ecosystems/c/celo.toml

```diff
@@ -58,13 +61,15 @@ sub_ecosystems = [
   "Plock Fi",
   "Poof-cash",
   "PrimeDAO",
+  "Pyth",
```

Comment:
Please remove this one, we are keeping it "General" to all crypto and not attaching it to any compatible ecosystem. This is because it has compatibility with almost all chains and in those cases we don't tag one by one